op-e2e/actions/interop/proofs_test.go

```diff
@@ -1293,21 +1293,24 @@ func TestInteropFaultProofs_DepositMessage_InvalidExecution(gt *testing.T) {
 	runFppAndChallengerTests(gt, system, tests)
 }
 
-func runFppAndChallengerTests(gt *testing.T, system *dsl.InteropDSL, tests []*transitionTest) {
+// Returns true if all tests passed, otherwise returns false
+func runFppAndChallengerTests(gt *testing.T, system *dsl.InteropDSL, tests []*transitionTest) bool {
+	passed := true
 	for _, test := range tests {
 		test := test
-		gt.Run(fmt.Sprintf("%s-fpp", test.name), func(gt *testing.T) {
+		passed = gt.Run(fmt.Sprintf("%s-fpp", test.name), func(gt *testing.T) {
 			runFppTest(gt, test, system.Actors, system.DepSet())
-		})
+		}) && passed
 
-		gt.Run(fmt.Sprintf("%s-challenger", test.name), func(gt *testing.T) {
+		passed = gt.Run(fmt.Sprintf("%s-challenger", test.name), func(gt *testing.T) {
 			runChallengerTest(gt, test, system.Actors)
-		})
+		}) && passed
 	}
+	return passed
 }
 
 func runFppTest(gt *testing.T, test *transitionTest, actors *dsl.InteropActors, depSet *depset.StaticConfigDependencySet) {
-	t := helpers.NewDefaultTesting(gt)
+	t := helpers.SubTest(gt)
```

Comment:
This disables t.Parallel(), preventing a race condition causing test failures that was occurring when calling `assertProgramOutputMatchesDerivationForBlockTimestamp()` in between mutations to the action test state.